interpreter/dune

```diff
@@ -47,11 +47,6 @@
  (menhir
   (modules parser)))
 
-(env
- (_
-  (flags
-   (-w +a-4-27-42-44-45-70 -warn-error +a-3))))
```

Comment:
Except for removing 27, I think we still want to keep this declaration for stricter checking? The default is much more lax, no?

Reply:
The default of the OCaml compiler are more lax. Dune has its own set of warnings which is much more stricter.

Reply:
Not stricter than this, AFAICT.